src/openxr/OpenXRApi.cpp

```diff
@@ -953,6 +960,21 @@ bool OpenXRApi::initialiseSession() {
 	return true;
 }
 
+bool OpenXRApi::set_render_target_size_multiplier(float multiplier) {
+	if (is_initialised()) {
+		Godot::print_error("Setting the render target size multiplier is only allowed prior to initialization.", __FUNCTION__, __FILE__, __LINE__);
```

Comment:
Probably not for now but I think we need to look into what it takes to recreate the swapchain if anything changes that requires a differently sized swap chain. 

SteamVR for instance has a mode where it will periodically tweak the size of the render target depending on system load, so if it finds out you're consistently not reaching steady FPS, it will request a differently sized render target.

OpenXR doesn't have an event to notify us of this yet and we could only implement this by polling but it's something to start thinking about. Again not for this PR but for later.

Also there is obviously a clash between XR runtimes that already support a multiplier as a feature and us now having our own that is applied on top of it but c'est la vie.

Reply:
Agreed.
Similar to that, we're also assuming that all the `XrViewConfigurationView` properties will have matching render target dimensions and swapchain sample count. This may not always be the case (e.g: devices with more than 2 views), so here also this is something we may need to address in the future.